src/backends/backend.js

```diff
@@ -51,11 +51,15 @@ const slugFormatter = (template = "{{slug}}", entryData) => {
       case "day":
         return (`0${ date.getDate() }`).slice(-2);
       case "slug":
-        return slug(getIdentifier(entryData).trim(), {lower: true});
+        return getIdentifier(entryData).trim();
       default:
-        return slug(entryData.get(field, "").trim(), {lower: true});
+        return entryData.get(field, "").trim();
     }
-  });
+  })
```

Comment:
Can you bring back your source comment explaining this regex?

Reply:
Done.